distribution/changelog.txt

```diff
@@ -1,18 +1,24 @@
 0.0.5 (in development)
 ------------------------------------------------------------------------
+- Feature: Add OpenGL drawing engine.
 - Feature: Ability to rotate map elements with the tile inspector.
 - Feature: Add cheat to disable littering.
 - Feature: Add ride console command for diagnostics and changing vehicle type.
 - Feature: Allow setting ownership of map edges.
 - Feature: Allow up to 255 cars per train.
 - Feature: Importing SV4 and SC4 files with rides.
-- Feature: Added Norwegian translation
+- Feature: Add Norwegian translation.
 - Feature: Cheat to disable plant aging.
+- Feature: Chain cheat that allows any piece to be made into a track piece.
```

Comment:
Thanks you, it was difficult for me to write it down correctly :) Will fix it tonight and waiting for more suggestions. :)